src/api/spec/components/comment_component_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+RSpec.describe CommentComponent, type: :component do
+  let(:builder) { Builder::XmlMarkup.new }
+
+  before do
+    render_inline(described_class.new(comment: comment, obj_is_user: true, builder: builder))
+  end
+
+  shared_examples 'rendering the body, who, when and the id' do
```

Comment:
I won't block a PR for this, but shared examples are a pain to update. Would it be possible to avoid using shared examples?

Reply:
It's simple enough now and it's not actually giving problems. Once it starts to be a real pain, we can inline it, but not now.